src/main/scala/com/gu/fastly/Lambda.scala

```diff
@@ -23,6 +23,60 @@ class Lambda {
   private val cloudWatchClient = AmazonCloudWatchClientBuilder.defaultClient
   private val snsClient = AmazonSNSClientBuilder.defaultClient
 
+  private def raiseAllThePurges(event: Event): Boolean = {
+    val contentType = extractUpdateContentType(event)
+    val purgeType = if (event.eventType == EventType.Delete) {
+      Hard
+    } else {
+      Soft
+    }
+    val aliasPaths = extractAliasPaths(event)
+    val dotcomCanonicalPurge = sendFastlyPurgeRequest(event.payloadId, purgeType, config.fastlyDotcomServiceId, makeDotcomSurrogateKey(event.payloadId), config.fastlyDotcomApiKey, contentType)
```

Comment:
Is `event.payloadId` the 'canonical' path?

Reply:
Yeah it's likely to be a contentId like /business/2010/meh